### ISSUE TITLE
Question: dN/dS of 10

### ISSUE THREAD

**Issue by zx8754** [im3sanger/dndscv#19]
From the manual:
http://htmlpreview.github.io/?http://github.com/im3sanger/dndscv/blob/master/vignettes/dNdScv.html

> Note in the table that the dN/dS ratios of significant cancer genes are typically extremely high, often >10 or even >100. This contains information about the fraction of mutations observed in a gene that are genuine drivers under positive selection. For example, for a highly significant gene, a **dN/dS of 10** indicates that there are 10 times more non-synonymous mutations in the gene than neutrally expected, suggesting that at least around 90% of these mutations are genuine drivers (Greenman et al, 2006; Martincorena et al, 2017).

What are you referring to with `"dN/dS of 10"`, where is this value in the outputs?

**Comment by zx8754** [comment]
Got some ELI5 from @jrjhealey (thank you):

> dN/dS: How many bases are changing to other amino acids versus how many remain the same amino acid.
> If the ratio is positive then the position is selected for positively (I think I have that the right way round)
> N = Nonsynonymous
> S = Synonymous
> If more mutations are S than N, then the site is being selected for, if it's N, then it's neutral or selected against.
> Essentially, dN/dS is just a measure of how selection pressure is acting at that site.


@jrjhealey, please add more info, if I got anything wrong.

**Comment by im3sanger** [comment]
Hello,

dN/dS ratios are represent the ratio between the non-synonymous and synonymous mutation rates normalised by neutral expectation given sequence composition and the observed trinucleotide substitution rates. When this normalisation is done correctly, dN/dS ratios significantly >1 reveal positive selection, and ratios <1 reveal negative selection.

Point estimates for the dN/dS ratios at gene level are available in the sel_cv output:
* wmis_cv: dN/dS for missense mutations
* wnon_cv: dN/dS for nonsense mutations
* wspl_cv: dN/dS for splice site mutations (note that, by default, a single parameter is used for nonsense and essential splice site mutations and so wnon_cv==wspl_cv. See documentation for more details and for information on how to change this default setting).
* wind_cv: observed/expected ratios (not quite dN/dS) for indels (and changes other than single nucleotide changes).

pvalues and qvalues for most of these mutation types are available as additional columns and inform about whether the dN/dS values are significantly different from 1. The global p and q values aggregate the information across mutation classes to provide a single q-value per gene.

dN/dS values can also be calculated at a whole-exome level, aggregating all mutations across genes. As described in the tutorial, these are available using dndsout$globaldnds.

For more details on dN/dS ratios, the mathematics behind and the types of conclusions that can be drawn from them in cancer, see the paper below:
https://linkinghub.elsevier.com/retrieve/pii/S0092-8674(17)31136-4

Inigo
